eng/pipelines/templates/stages/archetype-js-release.yml

```diff
@@ -121,8 +121,9 @@ stages:
                       - bash: |
                           npm install
                         displayName: Install versioning tool dependencies
+                        WorkingDirectory: ./eng/tools/versioning
                       - bash: |
-                          node ./eng/tools/versioning/increment.js --artifact-name ${{ artifact.name }} --repo-root .
+                          node ./increment.js --artifact-name ${{ artifact.name }} --repo-root .
```

Comment:
This command has no working directory specified so it'll probably fail because it's looking for `increment.js` in the root of the source folder

Reply:
fixed